app.py

```diff
@@ -1,6 +1,11 @@
 from flask import Flask
+import os
+
 app = Flask(__name__)
 
 @app.route('/')
 def hello_world():
     return 'Hello, World!'
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", port=int(os.environ.get("PORT", 6969)))
```

Comment:
host and port should be loaded from application's database:
https://app.clubhouse.io/yeoldpsic/story/53/configuration-settings-should-come-from-database